boms/user/ee/ee-with-tools/pom.xml

```diff
@@ -134,6 +134,10 @@
                                     <groupId>org.jboss.arquillian.protocol</groupId>
                                     <artifactId>arquillian-protocol-servlet-jakarta</artifactId>
                                 </dependency>
+                                <dependency>
+                                    <groupId>org.jboss.resteasy</groupId>
+                                    <artifactId>resteasy-undertow-cdi</artifactId>
+                                </dependency>                                
```

Comment:
We should not add this. This is for the Jakarta REST `SeBootstrap` which is for SE only environments and should not be used in a Jakarta EE Container.